panda/src/glstuff/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
+if(ANDROID)
+else()
 if(NOT HAVE_GL)
   return()
 endif()
+endif()
```

Comment:
`if(NOT ANDROID AND NOT HAVE_GL)`… actually do we want to pick this up when building GLES on other platforms as well?  So `if(NOT HAVE_GLES AND NOT HAVE_GLES2 AND NOT HAVE_GL)`?

Reply:
i've thought if(android) elif(webgl) elif(etc ... ) would be easier to read than a  big too much NOT HAVE_*  if at once